src/sidebars/search/search-controller.js

```diff
@@ -17,8 +18,35 @@
                     selectionService.select(stock);
                 };
 
+                self.onSearchKeyUp = function(event) {
+                    if (event.keyCode === 38) {
+                        // Up
+                        changePointer(-1);
+                    } else if (event.keyCode === 40) {
+                        // Down
+                        changePointer(1);
+                    }
+                };
+
+                function changePointer(delta) {
+                    var newPointer = self.stockSelectionPointer + delta;
+
+                    self.stockSelectionPointer = Math.max(
+                        0,
+                        Math.min(
+                            newPointer,
+                            self.stocks.length - 1
+                        )
+                    );
```

Comment:
It's limiting it so you can't try and set the pointer to something that doesn't exist . The search box retains focus throughout (similar to how most autocompletes work), all the arrow keys do is change the selected stock